openstack/02b-MAChigh/sixtop.c

```diff
@@ -631,7 +631,7 @@ owerror_t sixtop_send_internal(
 //        msg->l2_radioType = (radioType_t)(msg->l2_dsn&0x01);
 //    }
 
-    msg->l2_radioType = 0;
+    msg->l2_radioType = 1;

```

Comment:
This is a hack for debugging purpose, you can do this for now., It's better to think what the right way to determine which radioType to use.